spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -249,13 +249,13 @@ def create!(name: nil, bundle_id: nil, certificate: nil, devices: [], mac: false
         # @return (Array) Returns all profiles registered for this account
         #  If you're calling this from a subclass (like AdHoc), this will
         #  only return the profiles that are of this type
-        def all(mac: false)
+        def all(mac: false, xcode: false)
```

Comment:
Could you add a comment above this method on what the parameter does for future reference 👍